BREEZE.rst

```diff
@@ -233,6 +233,16 @@ Run this command to install Breeze (make sure to use ``-e`` flag):
 Once this is complete, you should have ``breeze`` binary on your PATH and available to run by ``breeze``
 command.
 
+**Docker errors that may come while running breeze**
```

Comment:
I think it would be better to mention those in the `Prerequisite` section. I think we should even add a step of verifying if docker works for you there.

Typically that can be done via `docker run hello-world`. And this is happening when all is configured:

```
docker run hello-world

Hello from Docker!
This message shows that your installation appears to be working correctly.

To generate this message, Docker took the following steps:
 1. The Docker client contacted the Docker daemon.
 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
    (arm64v8)
 3. The Docker daemon created a new container from that image which runs the
    executable that produces the output you are currently reading.
 4. The Docker daemon streamed that output to the Docker client, which sent it
    to your terminal.

To try something more ambitious, you can run an Ubuntu container with:
 $ docker run -it ubuntu bash

Share images, automate workflows, and more with a free Docker ID:
 https://hub.docker.com/

For more examples and ideas, visit:
 https://docs.docker.com/get-started/
```

BTW. Technically DOCKER_BUILDKIT=1 is not needed. Breeze automaticlaly add DOCKER_BUILDKIT=1 whenever it runs any `docker build` command. I wonder where you had to add it ?

Reply:
Actually i was having this error `if error response from dockerfile parse error`, even though everything was fine with docker,
when i added `DOCKER_BUILDKIT=1`, then everything was working fine, so i thought to mention in documentation.

Reply:
Do you remember which command you had problem with ?

Reply:
Or maybe you can try to reproduce it now by removing the env var and send me the output ?

Reply:
@potiuk 

![image](https://user-images.githubusercontent.com/120032754/222951007-369e4fbe-e16b-48a1-9329-8dd2573c5658.png)


I restarted laptop so DOCKER_BUILDKIT=1 this was not done, when i ran `breeze ci-image build --upgrade-to-newer-dependencies`
above error was coming, although docker run hello-world was working fine.

After doing export DOCKER_BUILDKIT=1, error is resolved, below is the result
![image](https://user-images.githubusercontent.com/120032754/222951097-2b713ebb-a2e2-4275-bdae-32076455bd88.png)

Reply:
Ah I see. I will take a look and fix it then :)

Reply:
Can you please run the build command with `--dry-run` (without the evn set) @Abhishek-kumar-samsung  ?

Reply:
OK. No need. Fix here: https://github.com/apache/airflow/pull/29928

Reply:
`breeze ci-image build --upgrade-to-newer-dependencies --dry-run`  this command?

This command is not giving any error.

This message comes at the end `Not updating build hash because we are in `dry_run` mode.`

Reply:
I am not sure if `DOCKER_BUILDKIT=1` this would be required all the time. So in documentation, i have written **_If_** the error comes, then we can try this thing.

Reply:
Ok, so should i remove that line from this PR?

Just put that one fix of `if docker not runing in python virtual environment`